rules/windows/create_stream_hash/create_stream_hash_file_sharing_domains_download_unusual_extension.yml

```diff
@@ -1,13 +1,16 @@
 title: Unusual File Download from File Sharing Domain
```

Comment:
```suggestion
title: Unusual File Download from WEB File Sharing Domain
```

Reply:
It's not a "web file sharing" it's a "file sharing" so can't use web here.

Reply:
I wil replace `Domain` with `Website` as discussed